backend/build.gradle.kts

```diff
@@ -54,6 +54,27 @@ tasks.test {
     finalizedBy(tasks.jacocoTestReport) // Generate the report after running tests
 }
 
+tasks.jacocoTestReport {
```

Comment:
This is for removing some of the untestable classes from our code coverage such as the Router classes and the MiscrosoftKeyset (Which is hard to test since it is basically a wrapper around a REST call to MS)